src/TesterInternal/General/BuiltInSerializerTests.cs

```diff
@@ -29,7 +29,7 @@ public class BuiltInSerializerTests
         public void InitializeForTesting()
         {
             TraceLogger.Initialize(new NodeConfiguration());
-            SerializationManager.Initialize(false, null);
+            SerializationManager.Initialize(false, null, false);
```

Comment:
I ran all tests with json serializer, so that i found the issues with it. Yes, I'll add new tests.